freeCodeCamp/freeCodeCamp
# feat(challenge-editor): add english/task helper scripts

This adds some helper scripts in the challenge editor for task style challenges.

To test:

Optional:
-run `pnpm challenge-editor` to start the challenge editor
-go to `localhost:3000` to open the challenge editor
-click on the "A2 English for Developers" superblock
-click on one of the blocks to see the challenges
-follow the instructions at the bottom to add/remove challenges - which say this...

Actual:
-navigate to `freeCodeCamp/curriculum/challenges/english/21-a2-english-for-developers/(block)/` in the terminal
-run `pnpm create-next-task` to create the next task style challenge in this block
-run `pnpm insert-task` to create a new task style challenge in the middle of this block
-run `pnpm delete-task` to delete a task style challenge in this block.
-run `pnpm reorder-tasks` to rename the tasks to the correct order

Checklist:






Closes #XXXXX
